plugins/destination/postgresql/client/write.go

```diff
@@ -3,127 +3,268 @@ package client
 import (
 	"context"
 	"fmt"
-	"sort"
 	"strings"
+	"sync/atomic"
 
+	"github.com/cloudquery/plugin-sdk/cqtypes"
+	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgtype"
 	"github.com/jackc/pgx/v4"
 )
 
-func (c *Client) Write(ctx context.Context, table string, data map[string]interface{}) error {
-	var sql string
-	var values []interface{}
+func CQTextArray(c *cqtypes.TextArray) pgtype.TextArray {
```

Comment:
nit: two things about these functions:
 - these functions look like they don't need to be public (but I could be wrong there), and
 - I think the name should be e.g. `pgTextArray` or `toPgTextArray`, since it's converting from CQ to PG, but the current name would suggest the opposite.

Reply:
Thanks! Changed to private and better Naming